app/views/shared/_failure.html.erb

```diff
@@ -6,7 +6,7 @@
   <%= render PageHeadingComponent.new.with_content(presenter.header) %>
 <% end %>
 
-<% Array(presenter.description).each do |description_p| %>
+<% Array(presenter.description(self)).each do |description_p| %>
```

Comment:
I don't like it, and we encountered it once before in https://github.com/18F/identity-idp/pull/6023#discussion_r817995755, but as noted in extended commit description of dcef56675 and original pull request comment text, we can't rely on `view_context` remaining constant between the time of presenter initialization and view rendering. The value of `view_context` in a constructed presenter may receive scripts to enqueue associated with the component, but those scripts would not be the same as rendered into the template. We sorta avoided this in #6023 because the presenter was constructed inside a helper, so view rendering had already begun, and the render-bound `view_context` was available.

I think an ideal long-term solution is to remove these presenters and views altogether, then create a shared "Failure" component that can take the place of this view and the [`idv/shared/_error.html.erb`](https://github.com/18F/identity-idp/blob/bd1ca8c3fd7f17b5df6d54b67bac26a65e0b8de9/app/views/idv/shared/_error.html.erb) partial, then each failure type can have a dedicated view that calls directly to the component.

e.g.

```erb
<%# app/views/shared/_otp_max_attempts') %>

<%= render FailurePageComponent.new(...) %>
```